cypress/integration/visual-test/ActivationCard-dark_spec.js

```diff
@@ -3,7 +3,7 @@ describe('ActivationCard dark mode visual regression check', () => {
     cy.visit('/visual-test/ActivationCard-dark');
   });
 
-  it('Compares screenshots', () => {
+  it('Compares snapshots', () => {
```

Comment:
This PR would be a lot easier to review if these changes had been made in a separate PR

Reply:
I only did this cos main change was only in a single page, so everything else could be skipped.